test/python/pulse/test_instruction_schedule_map.py

```diff
@@ -393,15 +393,15 @@ def test_schedule_with_non_alphanumeric_ordering(self):
         """Test adding and getting schedule with non obvious parameter ordering."""
         theta = Parameter("theta")
         phi = Parameter("phi")
-        lamb = Parameter("lambda")
+        lamb = Parameter("lam")
```

Comment:
Interesting. https://github.com/python/cpython/issues/92062 is what is requiring the change here. If I understand correctly, `InstrumentScheduleMap` is using `inspect.signature` and `inspect.Parameter` for convenience internally but the schedule parameters do not need to form a valid Python signature (I don't see the signature being returned out of `InstructionScheduleMap`). So we could change the internals if we wanted to allow `lambda` as a parameter. `lambda` seems like a common name to want to use, but I think with `BackendV2` we want to move away from `InstructionScheduleMap` any way, so it's probably not worth changing for Python 3.11.

Reply:
This is all Alonzo Church's fault